proposals/description/fip_60.ts

```diff
@@ -44,6 +44,51 @@ const fip_60: ProposalDescription = {
       ],
       description: 'Add FEI-DAI G-UNI to FeiRari'
     },
+    {
+      target: 'rariPool8Comptroller',
+      values: '0',
+      method: '_setPauseGuardian(address)',
+      arguments: ['{fuseGuardian}'],
+      description: 'Set Fuse pause guardian'
+    },
+    {
+      target: 'fuseGuardian',
+      values: '0',
+      method: '_setBorrowPausedByUnderlying(address,bool)',
+      arguments: ['{curveD3pool}', true],
+      description: 'Set d3 borrow paused'
+    },
+    {
+      target: 'fuseGuardian',
+      values: '0',
+      method: '_setBorrowPausedByUnderlying(address,bool)',
+      arguments: ['{curve3Metapool}', true],
+      description: 'Set Fei-3Crv borrow paused'
+    },
+    {
+      target: 'fuseGuardian',
+      values: '0',
+      method: '_setBorrowPausedByUnderlying(address,bool)',
+      arguments: ['{gUniFeiDaiLP}', true],
+      description: 'Set Gelato borrow paused'
+    },
+    {
+      target: 'rariPool8Comptroller',
+      values: '0',
+      method: '_setBorrowCapGuardian(address)',
+      arguments: ['{fuseGuardian}'],
+      description: 'Set Fuse borrow cap guardian'
+    },
+    {
+      target: 'fuseGuardian',
+      values: '0',
+      method: '_setMarketSupplyCapsByUnderlying(address[],uint256[])',
+      arguments: [
+        ['{curveD3pool}', '{curve3Metapool}', '{gUniFeiDaiLP}'],
+        ['25000000000000000000000000', '25000000000000000000000000', '2500000000000000000000000000']
```

Comment:
We should come up with a better way to spell out amounts than an inline string, it's bound to cause a mistake eventually

Reply:
can probably just do ethers weiperether

Reply:
I do think we're covered by invariant checking in the validate step